connectors/atlassian/install.js

```diff
@@ -7,14 +7,17 @@ const getTemplateFile = (fileName) => Fs.readFileSync(__dirname + `/template/${f
 const Sdk = require('@fusebit/add-on-sdk');
 
 module.exports = async (ctx) => {
+  if (!ctx.caller.permissions) {
```

Comment:
Side note for future: it seems like there's a consistent pattern where things like install, uninstall, etc require permissions but a few others do not.  Might be worth codifying that in the manager.

Reply:
Just to be clear: not proposing any change in this PR.  Mark this issue as resolved once you've read it :)

Reply:
Makes sense to make this a part of the manager, there is definitely a very clear pattern. I've filed a ticket to track